Models/Database/WalletDetail.cs

```diff
@@ -11,16 +11,26 @@ public class WalletDetail : BaseModel<Guid>
         public string FirstName { get; set; }
         [Encrypted]
         public string LastName { get; set; }
+        [Encrypted(true)]
+        public string MobileNumber { get; set; }
+        public bool isMyMobileNumber { get; set; }
```

Comment:
what do you think about the name `isMobileNumberOwner`?

Reply:
I don't like it very much, if you have another suggestion? It's there to indicate if the mobile number belongs to that person and not someone else